### PR TITLE
MPI: Identidad autopercibida

### PR DESCRIPTION
**Pull request by Fabio-Ramirez**
### Requerimiento

https://proyectos.andes.gob.ar/browse/MPI-407

### Funcionalidad desarrollada 

1. Se agrega acciones para filtrar por numero de identificacion y/o alias

### UserStories llegó a completarse

- [x] Si
- [ ] No

### Requiere actualizaciones en la base de datos

- [ ] Si
- [x] No

### CODE REVIEW

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9220](http://condorpiedra.andes.gob.ar:85/9220/mochawesome.html)
CYPRESS RUN: [8366](https://dashboard.cypress.io/projects/xr7gft/runs/8366/specs)
TEST START: 2023-07-17T13:39:01.364Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9222](http://condorpiedra.andes.gob.ar:85/9222/mochawesome.html)
CYPRESS RUN: [8368](https://dashboard.cypress.io/projects/xr7gft/runs/8368/specs)
TEST START: 2023-07-17T16:11:57.258Z
TOTAL: 403
SUCCESS: 392
FAIL: 9
SKIPPED: 2

File: modules/rup/routes/prestacion.ts [hunk 1]
```diff
@@ -305,7 +305,7 @@ router.get('/prestaciones/solicitudes', async (req: any, res, next) => {
         };
 
         if (req.query.paciente && !Types.ObjectId.isValid(req.query.paciente)) {
-            (project.$project as any).datosPaciente = { $concat: ['$paciente.nombre', ' ', '$paciente.apellido', ' ', '$paciente.documento'] };
+            (project.$project as any).datosPaciente = { $concat: ['$paciente.nombre', ' ', '$paciente.apellido', ' ', '$paciente.documento',' ', '$paciente.numeroIdentificacion',' ', '$paciente.alias'] };
```
Comment (MarianoCampetella) [inline modules/rup/routes/prestacion.ts:308]: Para evitar el uso del as any podemos remplazarlo por
project.$project['datosPaciente'] = { ... };

**Review by MarianoCampetella — CHANGES_REQUESTED** [review]

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9237](http://condorpiedra.andes.gob.ar:85/9237/mochawesome.html)
CYPRESS RUN: [8383](https://dashboard.cypress.io/projects/xr7gft/runs/8383/specs)
TEST START: 2023-07-21T11:41:16.498Z
TOTAL: 403
SUCCESS: 391
FAIL: 10
SKIPPED: 2

File: modules/rup/routes/prestacion.ts (hunk 1 above)
Comment (Fabio-Ramirez) [reply modules/rup/routes/prestacion.ts]: Corregido compa!

**Review by MarianoCampetella — APPROVED** [review]

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9255](http://condorpiedra.andes.gob.ar:85/9255/mochawesome.html)
CYPRESS RUN: [8401](https://dashboard.cypress.io/projects/xr7gft/runs/8401/specs)
TEST START: 2023-07-24T18:25:26.569Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9256](http://condorpiedra.andes.gob.ar:85/9256/mochawesome.html)
CYPRESS RUN: [8402](https://dashboard.cypress.io/projects/xr7gft/runs/8402/specs)
TEST START: 2023-07-24T18:37:05.467Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9285](http://condorpiedra.andes.gob.ar:85/9285/mochawesome.html)
CYPRESS RUN: [8430](https://dashboard.cypress.io/projects/xr7gft/runs/8430/specs)
TEST START: 2023-07-26T18:38:58.532Z
TOTAL: 341
SUCCESS: 334
FAIL: 5
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9287](http://condorpiedra.andes.gob.ar:85/9287/mochawesome.html)
CYPRESS RUN: [8432](https://dashboard.cypress.io/projects/xr7gft/runs/8432/specs)
TEST START: 2023-07-26T19:07:57.248Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9288](http://condorpiedra.andes.gob.ar:85/9288/mochawesome.html)
CYPRESS RUN: [8433](https://dashboard.cypress.io/projects/xr7gft/runs/8433/specs)
TEST START: 2023-07-26T19:33:11.951Z
TOTAL: 342
SUCCESS: 329
FAIL: 11
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9296](http://condorpiedra.andes.gob.ar:85/9296/mochawesome.html)
CYPRESS RUN: [8441](https://dashboard.cypress.io/projects/xr7gft/runs/8441/specs)
TEST START: 2023-07-27T15:56:11.023Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9297](http://condorpiedra.andes.gob.ar:85/9297/mochawesome.html)
CYPRESS RUN: [8442](https://dashboard.cypress.io/projects/xr7gft/runs/8442/specs)
TEST START: 2023-07-27T16:08:19.463Z
TOTAL: 403
SUCCESS: 400
FAIL: 1
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9299](http://condorpiedra.andes.gob.ar:85/9299/mochawesome.html)
CYPRESS RUN: [8444](https://dashboard.cypress.io/projects/xr7gft/runs/8444/specs)
TEST START: 2023-07-27T18:38:05.685Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9328](http://condorpiedra.andes.gob.ar:85/9328/mochawesome.html)
CYPRESS RUN: [8469](https://dashboard.cypress.io/projects/xr7gft/runs/8469/specs)
TEST START: 2023-08-02T11:43:21.208Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9329](http://condorpiedra.andes.gob.ar:85/9329/mochawesome.html)
CYPRESS RUN: [8470](https://dashboard.cypress.io/projects/xr7gft/runs/8470/specs)
TEST START: 2023-08-02T16:09:35.416Z
TOTAL: 312
SUCCESS: 294
FAIL: 16
SKIPPED: 2
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9330](http://condorpiedra.andes.gob.ar:85/9330/mochawesome.html)
CYPRESS RUN: [8471](https://dashboard.cypress.io/projects/xr7gft/runs/8471/specs)
TEST START: 2023-08-02T16:35:19.896Z
TOTAL: 403
SUCCESS: 399
FAIL: 2
SKIPPED: 2

**Comment by silviroa** [comment]
USUARIO: framirez  
BUILD NUMBER: [9331](http://condorpiedra.andes.gob.ar:85/9331/mochawesome.html)
CYPRESS RUN: [8472](https://dashboard.cypress.io/projects/xr7gft/runs/8472/specs)
TEST START: 2023-08-02T18:48:38.522Z
TOTAL: 403
SUCCESS: 401
FAIL: 0
SKIPPED: 2

**Review by negro89 — APPROVED** [review]
codigo ok

**Pull request merged**